### DISCUSSION TITLE
[V5] make custom message for error handling in confirmed rule

### DISCUSSION THREAD

**Discussion by AhmedHashem2104** [adonisjs/core#2200] [Help]
i want to customize the message for confirmed rule in adonisjs v5

**Comment by adufr** [comment]
Maybe this will help?
https://preview.adonisjs.com/guides/validator/custom-messages

**Reply by AhmedHashem2104** [reply]
i can't custom message on password confirmation , give me the code pls

**Reply by hiteshrsdigitech** [reply]
Try this

```
messages: {
    'password.confirmed': 'Your message'
}
```
